plugin.yaml

```diff
@@ -348,6 +352,34 @@ node_types:
         resize:
           implementation: vsphere.vsphere_storage_plugin.storage.resize
 
+  cloudify.vsphere.nodes.ISO9660:
+    derived_from: cloudify.nodes.Volume
+    properties:
+      datacenter_name:
+        description: >
+          datacenter name.
+        type: string
+        required: false
+      allowed_datastores:
+        description: >
+          Which ESX datastore(s) this server is allowed to be deployed on.
+        required: false
+      files:
+        default: {}
+        description: >
+          File list with content. (Key:value)
```

Comment:
I'm not 100% clear on this- does it mean that this is the list of files to go on the virtual CDROM?
If so, maybe it should read something like:
description: >
    List of files on CD with their contents, e.g.
        README: >
              This is the readme for the new CD image.
               This will be visible when the CD image is mounted.
         data: This is a second file on the CD image.

Reply:
I will change